projects/js-packages/publicize-components/src/components/settings-button/index.js

```diff
@@ -6,7 +6,7 @@
  * If window/tab is closed,
  * then connections will be automatically refreshed.
  */
-import { getSiteFragment, getJetpackData } from '@automattic/jetpack-shared-extension-utils';
+import { usePublicizeConfig } from '@automattic/jetpack-publicize-components';
```

Comment:
I think `usePublicizeConfig` is local to this component now, so we could import it as `../../hooks/use-publicize-config`

Reply:
Done!